src/pcrlock/pcrlock.c

```diff
@@ -5133,6 +5234,34 @@ static int run(int argc, char *argv[]) {
         if (r <= 0)
                 return r;
 
+        if (arg_varlink) {
+                _cleanup_(varlink_server_unrefp) VarlinkServer *varlink_server = NULL;
+
+                /* Invocation as Varlink service */
+
+                r = varlink_server_new(&varlink_server, VARLINK_SERVER_ROOT_ONLY);
```

Comment:
What about polkit integration? At least for reading logs it seems useful to have?

Reply:
Yeah, but let's add that later. We can open this up if we want to.

It's not entirely clear why we would not just make it world readable though, which is why i didn't add this yet.

Reply:
Ok, can you list it in the TODO so we don't forget?